ros/app.py

```diff
@@ -2,9 +2,9 @@
 from flask import Flask
 from flask_restful import Api
 from flask_cors import CORS
-from ros.models import db
-from ros.config import DB_URI
-from ros.resources.routes import initialize_routes
+from ros.lib.models import db
+from ros.lib.config import DB_URI
+from ros.api.routes import initialize_routes
 
 app = Flask(__name__)
```

Comment:
I think we can delete this `app.py` file.  Once this PR is merged it will be a dead code.

Reply:
You're right, this is a merge error.